mnefun/_mnefun.py

```diff
@@ -2330,3 +2332,114 @@ def plot_reconstruction(evoked, origin=(0., 0., 0.04)):
     axs[2, 0].set_ylabel('Reconstruction')
     fig.tight_layout()
     return fig
+
+
+def plot_chpi_snr_raw(raw, winlen, nharm, stop=None):
+    """Compute and plot cHPI SNR from raw data
+
+    Parameters
+    ----------
+    winlen : int
```

Comment:
this should probably be `duration` in seconds, and convert it to samples internally